packages/react/src/components/Layer/index.tsx

```diff
@@ -57,13 +62,26 @@ const Layer = React.forwardRef<
   } & React.HTMLAttributes<HTMLDivElement>
 >(
   (
-    { as, className: customClassName, children, level: overrideLevel, ...rest },
+    {
+      as,
+      className: customClassName,
+      children,
+      level: overrideLevel,
+      withBackground,
```

Comment:
```suggestion
      withBackground = false,
```
Do we want to set the default to false, just noticed it was missing from the prop table on the overview page